### PR TITLE
[v5.x] - Expose PropertyName on Csla.Blazor.IPropertyInfo interface.

### LINKED ISSUE #2165: Expose PropertyName on Csla.Blazor.IPropertyInfo interface.
Currently unable to add a unique `id=""` attribute to HTML elements as `PropertyName` on the `Csla.Blazor.IPropertyInfo` interface is not exposed.

### PR DESCRIPTION
**Pull request by adrianwright109**
Fixes #2165

### CODE REVIEW

File: Source/Csla.Blazor/PropertyInfo.cs [hunk 1]
```diff
@@ -21,10 +21,6 @@ public class PropertyInfo : IPropertyInfo, INotifyPropertyChanged
     /// Gets the model
     /// </summary>
     protected object Model { get; }
-    /// <summary>
-    /// Gets the property name
-    /// </summary>
-    protected string PropertyName { get; }
```
Comment (rockfordlhotka) [inline Source/Csla.Blazor/PropertyInfo.cs:27]: There's no implementation here to actually get and return the property name, so how would this work?

I'd expect an implementation similar to the one in the `FriendlyName` property.

**Review by rockfordlhotka — CHANGES_REQUESTED** [review]
I think the property needs an implementation.

File: Source/Csla.Blazor/PropertyInfo.cs (hunk 1 above)
Comment (adrianwright109) [reply Source/Csla.Blazor/PropertyInfo.cs:27]: Isn't it getting set in the `PropertyInfo` constructor?
https://github.com/MarimerLLC/csla/blob/5ab18303d4a0c9ae218edfcbb6e64e15dd72b568/Source/Csla.Blazor/PropertyInfo.cs#L33

Comment (rockfordlhotka) [reply Source/Csla.Blazor/PropertyInfo.cs:27]: Oh, my bad!

**Review by rockfordlhotka — APPROVED** [review]

**Pull request merged**